### ISSUE TITLE
The font in terminal changes every time when I open gdbgui

### ISSUE THREAD

**Issue by yuhany1024** [cs01/gdbgui#298]
Hi,

I'm using Win10 Linux subsystem Ubuntu 18. Before I open gdbgui, the terminal is like:
![image](https://user-images.githubusercontent.com/35322222/67154117-ed3beb80-f2aa-11e9-8b03-5d11ec9bcc71.png)

Once I open gdbgui, the font changes automatically and looks very ugly:
![image](https://user-images.githubusercontent.com/35322222/67154126-22e0d480-f2ab-11e9-875a-0d61439754f7.png)

How can I solve the issue? Thanks!

**Comment by bjorn3** [comment]
How is it even possible for a program to change the font of a terminal emulator? Only the user should be able to change the font.

**Comment by cs01** [comment]
@bjorn3 I agree this seems unexpected, but maybe in Windows it's possible. I did a quick search and found https://stackoverflow.com/questions/3592673/change-console-font-in-windows, so it is technically possible for a program to change the font.

That said, this is certainly not something gdbgui is doing. It might somehow be done in one of the dependencies of gdbgui. These are the packages that are installed and used by gdbgui:
```
Click==7.0
Flask==0.12.4
Flask-Compress==1.4.0
Flask-SocketIO==2.9.6
gdbgui==0.13.2.0
gevent==1.4.0
greenlet==0.4.15
itsdangerous==1.1.0
Jinja2==2.10.3
MarkupSafe==1.1.1
pkg-resources==0.0.0
pygdbmi==0.9.0.2
Pygments==2.4.2
python-engineio==3.9.3
python-socketio==4.3.1
six==1.12.0
Werkzeug==0.16.0
```

I don't know specifically what is causing it to behave this way. What you might want to do is insert a breakpoint in the Python code with an interactive debugger like pdb or ipdb, and step through the code line-by-line until you can pin down the exact line that modifies the font in your terminal. Depending on which line is causing the issue, we can then consider next steps on how to fix the issue.

**Comment by bjorn3** [comment]
> I did a quick search and found https://stackoverflow.com/questions/3592673/change-console-font-in-windows, so it is technically possible for a program to change the font in Windows.

I think that would only work when using real Windows programs, and not Linux programs with WSL.

**Comment by cs01** [comment]
That might be the case. I don't use either so I don't know.
